src/parser/shared/modules/spells/bfa/azeritetraits/WoundBinder.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import Analyzer from 'parser/core/Analyzer';
+import SPELLS from 'common/SPELLS/index';
+import TraitStatisticBox, { STATISTIC_ORDER } from 'interface/others/TraitStatisticBox';
+import { calculateAzeriteEffects } from 'common/stats';
+import { formatNumber } from 'common/format';
+
+const woundBinderStats = traits => Object.values(traits).reduce((obj, rank) => {
+  const [haste] = calculateAzeriteEffects(SPELLS.WOUNDBINDER.id, rank);
+  obj.haste += haste;
+  return obj;
+}, {
+  haste: 0,
+});
+
+const BASE_HASTE_AMOUNT = .34;
+
+/**
+ Your healing effects have a chance to increase your Haste by up to 435 for 6 sec. Healing lower health targets will grant you more Haste.
+
+ This trait scales linearly starting at with 34% of the haste at 0% health and 100% at 100% health.
```

Comment:
So this is pure speculation at this point?

Reply:
If it is I would at least add something along the lines of:
"Values are base on tc in the priest discord because no game data is available".

Reply:
Done